### PR TITLE
add Daki

### PR DESCRIPTION
**Pull request by coralieco**
image

### CODE REVIEW

File: app/javascript/components/Icon.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ import Anchor from 'images/anchor.png'
 import Rocks from 'images/rocks.png'
 import Island from 'images/island.png'
 import './Icon.scss'
+import Drop from 'images/drop.png'
+import Add from 'images/add.png'
+import KeepIt from 'images/keepit.png'
+import Idea from 'images/idea.png'
```
Comment (hchevalier) [inline app/javascript/components/Icon.js:26]: Could you move these above the `import './Icon.scss'` please? :)

File: app/javascript/components/retrospectives/Daki.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    background-color: #d8969a;
+  }
+
+  #add {
+    background-color: #98b2d8;
+  }
+
+  #keep {
+    background-color: #b9d0a0;
+  }
```
Comment (hchevalier) [inline app/javascript/components/retrospectives/Daki.scss:26]: Could you just find colors that better match the icon ones for Add & Keep?

Also, the "Add" icon seems to have a different style (the 3 others have kind of an "ink" feeling).

Last question, did you design the icons? If yes, kudos! If not and you find them on a specific website, could you update the README to credit the authors in the dedicated section?

Thanks a lot!

**Review by hchevalier — CHANGES_REQUESTED** [review]
Really minor feedback to keep some consistency between icons and zone colors ;)

File: app/javascript/components/retrospectives/Daki.scss (hunk 2 above)
Comment (coralieco) [reply app/javascript/components/retrospectives/Daki.scss:26]: Yes sure, I was thinking that my color choices was not really "accessible". I will choose color with more contrasts, sorry about that. 

I didn't design the icons, I use flaticon! Of course I'll update the credits.

Comment (coralieco) [reply app/javascript/components/retrospectives/Daki.scss:26]: I changed the colors to better fit the icons. All combinaisons are accessible. What do you think? (the blue button "I'm done" could be improved for accessibility in later PRs)

image

Comment (coralieco) [reply app/javascript/components/retrospectives/Daki.scss:26]: Changed the add icon so it would be more ink-ed-styled

image

Comment (coralieco) [reply app/javascript/components/retrospectives/Daki.scss]: (PS: I left the green color for the add button, I find green and yellow  to fit quite well but I can change if you prefer)

**Review by hchevalier — APPROVED** [review]

**Pull request merged**